### ISSUE TITLE
Non-existent image on boot

### ISSUE THREAD

**Issue by swat30** [CloudBrewery/docrane#15]
When starting docrane, a container with a non-existent image still raises an error:

```
Traceback (most recent call last):
  File "/root/.virtualenvs/docrane/bin/docrane", line 10, in <module>
    sys.exit(main())
  File "/root/.virtualenvs/docrane/local/lib/python2.7/site-packages/docrane/main.py", line 80, in main
    run(key_dir)
  File "/root/.virtualenvs/docrane/local/lib/python2.7/site-packages/docrane/main.py", line 45, in run
    cont.ensure_running()
  File "/root/.virtualenvs/docrane/local/lib/python2.7/site-packages/docrane/container.py", line 73, in ensure_running
    self.create()
  File "/root/.virtualenvs/docrane/local/lib/python2.7/site-packages/docrane/container.py", line 84, in create
    util.create_docker_container(self.name, self.docker_params)
  File "/root/.virtualenvs/docrane/local/lib/python2.7/site-packages/docrane/util.py", line 127, in create_docker_container
    name=name)
  File "/root/.virtualenvs/docrane/local/lib/python2.7/site-packages/docker/client.py", line 526, in create_container
    return self.create_container_from_config(config, name)
  File "/root/.virtualenvs/docrane/local/lib/python2.7/site-packages/docker/client.py", line 534, in create_container_from_config
    return self._result(res, True)
  File "/root/.virtualenvs/docrane/local/lib/python2.7/site-packages/docker/client.py", line 93, in _result
    self._raise_for_status(response)
  File "/root/.virtualenvs/docrane/local/lib/python2.7/site-packages/docker/client.py", line 89, in _raise_for_status
    raise errors.APIError(e, response, explanation=explanation)
docker.errors.APIError: 404 Client Error: Not Found ("No such image: localhost:5050/openstack/quantum-server:turned-off (tag: turned-off)")
```

**Comment by swat30** [comment]
Looks like this bug only affects the 1.12 API branch, which hasn't been updated.

**Comment by swat30** [comment]
We're not going to backport any non-critical fixes to v1.12
